lib/gh-badges.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const { makeBadge } = require('./make-badge')
+const { PDFKitTextMeasurer, QuickTextMeasurer } = require('./text-measurer')
+
+class BadgeFactory {
+  constructor({ fontPath, fallbackFontPath, cache = true }) {
+    this.measurer = cache
+      ? new QuickTextMeasurer(fontPath, fallbackFontPath)
+      : new PDFKitTextMeasurer(fontPath, fallbackFontPath)
+  }
+
+  create(format) {
```

Comment:
It would be nice to have a docblock here with `@param` and `@return` types

Reply:
Good suggestion. Added in cb94899